src/pages/Farm/RewardAmounts.tsx

```diff
@@ -31,7 +31,7 @@ export const RewardAmounts = ({ foxAmount }: TRewardAmounts) => {
         <>
           <Text color='gray.500'>Available Rewards</Text>
           <Text fontSize='5xl' fontWeight='bold' mt='-10px' mb='-5px'>
-            ${fiatAmount}
+            ${fiatAmount || '0.00'}
```

Comment:
fiatAmount was an empty string when there where no available rewards

Reply:
I believe the idea for it to be blank was based on the fact that the `useCoinCapPrice` hook could fail to get the price but we would still be able to show their fox amount without the user seeing $0 and getting nervous

Reply:
to your point though maybe the `useRealTimeRewardAmounts` hook should return 0 as long as there is a price